src/components/Categories.js

```diff
@@ -1,7 +1,16 @@
-import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { CheckStatus } from '../redux/categories/categories';
 
-const Categories = () => (
-  <button className="check-status" type="button">Check Status</button>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  const categories = useSelector((state) => state.category);
+  console.log(categories);
```

Comment:
[OPTIONAL]
- It would be awesome if you removed this console statement. It makes the code look unclean 👍